scipy/integrate/_ivp/base.py

```diff
@@ -79,7 +77,21 @@ class OdeSolver:
         Boundary time --- the integration won't continue beyond it. It also
         determines the direction of the integration.
     vectorized : bool
-        Whether `fun` is implemented in a vectorized fashion.
+        Whether `fun` can be called in a vectorized fashion. Default is False.
+
+        If ``vectorized`` is False, `fun` will always be called with ``y`` of
+        shape ``(n,)``, where ``n = len(y0)``.
+
+        If ``vectorized`` is True, `fun` may be called with ``y`` of shape
+        ``(n, k)``, where ``k`` is an integer. In this case, `fun` must behave
+        such that ``fun(t, y)[:, i] == fun(t, y[:, i])`` (i.e. each column of
+        the result is the time derivative of the state corresponding with a
+        column of ``y``).
+
+        Setting ``vectorized=True`` allows for faster finite difference
+        approximation of the Jacobian by methods 'Radau` and 'BDF', but
+        will result in slower execution for other methods and for 'Radau' and
```

Comment:
What about the phrasing: "and can also negatively impact"

Reply:
```suggestion
        will result in slower execution for other methods
        and can also negatively impact 'Radau' and
```

Reply:
How about I split the sentence?

>      will result in slower execution for other methods. It can also
>      result in slower overall execution for 'Radau' and 'BDF' in some
>      circumstances (e.g. small ``len(y0)``).

Reply:
Yes, that reads well.